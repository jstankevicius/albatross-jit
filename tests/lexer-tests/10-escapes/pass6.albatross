# some strings have the string character
"strings use the \" character"
"like \"this\" string"
