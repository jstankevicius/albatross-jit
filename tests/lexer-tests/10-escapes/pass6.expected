1 2 STRING 27 strings use the " character
1 3 STRING 18 like "this" string
